### PR TITLE
Feature/upload book

### PR DESCRIPTION
**Pull request by dhp1211**
Add 'create new book' page.

### CODE REVIEW

File: bookshelf-KP/app/controllers/books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def destroy
+  end
+
+  def show
+  end
+
+  private
+
+  def book_params
+    params.require(:book).permit(:title, :discription, :attachment, :cover)
```
Comment (vinhnglx) [inline bookshelf-KP/app/controllers/books_controller.rb]: same as @ntkien92

File: bookshelf-KP/app/assets/stylesheets/books.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// Place all the styles related to the Books controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+@import "bootstrap";
```
Comment (vinhnglx) [inline bookshelf-KP/app/assets/stylesheets/books.scss]: only import bootstrap ONE TIME

File: bookshelf-KP/app/views/books/create.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (vinhnglx) [inline bookshelf-KP/app/views/books/create.html.erb]: why you push this file @

File: bookshelf-KP/app/controllers/concerns/book/parameter.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+module Book::Parameter
+  extend ActiveSupport::Concern
+  private
```
Comment (vinhnglx) [inline bookshelf-KP/app/controllers/concerns/book/parameter.rb]: Add blank line @dhp1211

File: bookshelf-KP/app/models/book.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class Book < ActiveRecord::Base
+  mount_uploader :cover, CoverUploader
```
Comment (vinhnglx) [inline bookshelf-KP/app/models/book.rb]: blank line

**Comment by vinhnglx** [comment]
@dhp1211 shouln't push public folder to GH

**Comment by vinhnglx** [comment]
@dhp1211 please use `.gitignore` for public folder

**Comment by dhp1211** [comment]
@vinhnglx What folder are you talking about? I have no idea.

**Pull request merged**